container-chains/nodes/frontier/src/service.rs

```diff
@@ -438,6 +438,7 @@ pub async fn start_dev_node(
                     let time = MockTimestampInherentDataProvider;
                     let mocked_parachain = MockValidationDataInherentDataProvider {
                         current_para_block,
+                        current_para_block_head: None, // TODO
```

Comment:
@girazoki do you know if it's safe to set this to None?

Reply:
not sure but this is the dev service, so I would say lets keep it if it works

Reply:
disregard